### ISSUE TITLE
selenium/javascript/node/selenium-webdriver/example/google_search_test.js fails

### ISSUE THREAD

**Issue by joma74** [SeleniumHQ/selenium#3816]
## Meta -
OS:  Ubuntu 14.04
Selenium Version:  
selenium-webdriver@3.3.0
Browser:  
Google Chrome 57.0.2987.133
Browser Version:  
57.0.2987.133 (64-bit)
Node:
6.5.0, 6.9.4, 7.5.0

## Expected Behavior -

Beginners examples selenium/javascript/node/selenium-webdriver/example/google_search_test.js pass

## Actual Behavior -

it 'works with promises' fails

## Steps to reproduce -

First - Hello from Europe/Austria(not Australia, ahaha :bowtie: ). It is a localisation problem, but nevertheless very distracting for a selenium beginner.

First let me quote https://support.google.com/chrome/answer/173424?visit_id=0-636275415019771099-1854089045&rd=1
> Change the language of your Chrome browser (Windows & Chromebook only)
You can set Chrome to show all settings and menus in the language you want. This option is only available on Windows and Chromebook computers.
On Mac or Linux? Chrome will automatically display in the default system language for your computer.
...

Hm, why, but anyway ...

```bash
joma@kopernikus-u:~/entwicklung/nodews/vuejs-laracasts/episode-selenium-basics/node_modules$ cat /etc/default/locale
LANG="en_US.UTF-8"
LC_NUMERIC="de_AT.UTF-8"
LC_TIME="de_AT.UTF-8"
LC_MONETARY="de_AT.UTF-8"
LC_PAPER="de_AT.UTF-8"
LC_NAME="de_AT.UTF-8"
LC_ADDRESS="de_AT.UTF-8"
LC_TELEPHONE="de_AT.UTF-8"

LC_MEASUREMENT="de_AT.UTF-8"
LC_IDENTIFICATION="de_AT.UTF-8"
```
Dunno why, but google/chrome ignores my very LANG. Other story.

`it('works with generators')` passes, as it uses ncr -> http://www.google.com/ncr. ncr is no-country-redirection. So `until.titleIs` stays the expected english. 

`it('works with promises')` does use `https://www.google.com/`(no `/ncr`), so gets `de` localized, then  `until.titleIs` turns "webdriver - Google-Suche".

TL;DR; use `http://www.google.com/ncr` to make these tests work globally.

P.S. Executing ```LANG="en_US.UTF-8" mocha -t 20000 selenium-webdriver/example/google_sea
rch_test.js``` passes even without ncr. 

P.S.2 ```LANG="en_US.UTF-8" mocha -t 20000 selenium-webdriver/example/google_sea
rch_test.js``` does not work anymore. Chrome UI **is** localized to english(settings are in english, notification bar is `Chrome is being controlled by automated test software`). But as of now Google redirects me to `google.at`, although my test requests `http://www.google.com/`. Hence the title turns  german again. :expressionless:
